app/controllers/form_submissions_controller.rb

```diff
@@ -1,5 +1,3 @@
 class FormSubmissionsController < ApplicationController
-  def edit
-    head :ok
-  end
+  def index; end
```

Comment:
Not sure you need this - if you have an index view and it doesnt need anything special from the controller, I think you can leave out the method in the controller

Reply:
Removed in d5c894d but added back in 4f43bf8 as this method passes the reference and submission date to the view.